ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/AddParensToTypeTestFix.fs

```diff
@@ -0,0 +1,32 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Psi.ExtensionsAPI.Tree
+open JetBrains.ReSharper.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+
+type AddParensToTypeTestFix(error: RuntimeCoercionSourceSealedError) =
+    inherit FSharpQuickFixBase()
+
+    let typeTestExpr = error.Expr.As<ITypeTestExpr>()
+    let prefixApp = if isNotNull typeTestExpr then typeTestExpr.Expression.As<IPrefixAppExpr>() else null
+    let testedExpr = if isNotNull prefixApp then prefixApp.ArgumentExpression else null
+
+    override x.Text = "Add parens to type test"
```

Comment:
To be honest, this text does not seem clear enough to me. Maybe you have better ideas?

Reply:
Not yet, unfortunately, although I don't like it much either.